cmd/neofs-node/config/node/config.go

```diff
@@ -0,0 +1,77 @@
+package nodeconfig
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+	"github.com/nspcc-dev/neofs-node/pkg/network"
+)
+
+const (
+	subsection = "node"
+
+	KeyDefault = ""
+
+	attributePrefix = "attribute"
+)
+
+var ErrEmptyAddress = errors.New("empty bootstrap address, see `node.address` section")
+
+// Key returns value of "key" config parameter
+// from "node" section.
+//
+// Returns KeyDefault if value is not set.
+func Key(c *config.Config) string {
```

Comment:
It must panic if value is not a non-empty string.

Reply:
fixed, adapted commentary